lte/gateway/release/build-magma.sh

```diff
@@ -126,11 +126,17 @@ OAI_DEPS=(
     "liblfds710"
     "magma-sctpd >= ${SCTPD_MIN_VERSION}"
     "libczmq-dev >= 4.0.2-7"
+    "oai-gtp >= 4.9.5"
     )
 
 # OVS runtime dependencies
 OVS_DEPS=(
     "magma-libfluid >= 0.1.0.4"
+    "openvswitch-common >= 2.8.9"
+    "libopenvswitch >= 2.8.9"
+    "openvswitch-switch >= 2.8.9"
+    "openvswitch-datapath-source-4.9.0-9 >= 2.8.9"
+    "openvswitch-datapath-dkms >= 2.8.9"
```

Comment:
@tmdzk can we just install openvswitch-datapath-module-4.9.0-9-amd64_2.8.9-1_amd64.deb, 
Installing DKMS for OVS does not work since it depends on OAI-GTP which is non DKMS.